Makefile

```diff
@@ -54,13 +54,13 @@ BASE_IMAGE ?= gcr.io/distroless/static:nonroot
 BUILDER_IMAGE ?= golang:$(GO_VERSION)
 
 # ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
-ENVTEST_K8S_VERSION = 1.26
+ENVTEST_K8S_VERSION = 1.27
```

Comment:
Maybe we should bump the k8s to 1.27 the same time.

Reply:
Maybe keep the default versions in sync with the deps (1.26)
https://github.com/kubernetes-sigs/kueue/blob/e8209f25f047f299929b9c349f4ee80f4d120ca5/go.mod#L16-L21

Reply:
I think it's time to update the deps, rather :)

Reply:
We can hold this as is until we update the deps.

Reply:
> I think it's time to update the deps, rather :)

+1